### ISSUE TITLE
Consider referencing element classes in the HTML specification

### ISSUE THREAD

**Issue by jonathanKingston** [WICG/sanitizer-api#69]
As part of the spec, large numbers of elements are referenced in the specification that are generated with a script.

I might make sense even in a non normative sense to reference which elements / attributes are excluded / included from the default / baseline.

If this can be done at least partially by referencing [element kinds](https://html.spec.whatwg.org/#kinds-of-content) and deprecated features in the HTML specification this would make reading the specification intent easier.

**Comment by otherdaniel** [comment]
Partially resolved by #78.

**Comment by annevk** [comment]
This will be solved as part of merging it into the HTML Standard, no?

**Comment by mozfreddyb** [comment]
Agreed. I think we can close this. It's unlike we'll merge this without, isn't it?

**Comment by annevk** [comment]
Yeah, fair.
